src/components/Capture/index.js

```diff
@@ -14,15 +14,21 @@ import { functionalSwitch, impurify } from '../utils'
 import { canvasToBase64Images } from '../utils/canvas.js'
 import { fileToBase64, isOfFileType, fileToLossyBase64Image } from '../utils/file.js'
 
-export const DocumentCapture = options =>
```

Comment:
there is an utility called `impurify(statelessComponent)` that will wrap the stateless component into a stateful one. Btw, we should mark this code as a #HACK or #FIXUP, so that we keep track of this workaround.